packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -689,6 +698,40 @@ class CameraController extends ValueNotifier<CameraValue> {
     }
   }
 
+  /// Set the video stabilization mode for the selected camera.
+  ///
+  /// On Android (when using camera_android_camerax) and on iOS
+  /// the supplied [mode] value should be a mode in the list returned
+  /// by [getVideoStabilizationSupportedModes].
+  ///
+  /// Throws a [CameraException] when a not supported video stabilization
```

Comment:
nit: I think this is a bit clearer

```suggestion
  /// Throws a [CameraException] when an unsupported video stabilization
```